eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecs.scala

```diff
@@ -317,6 +317,8 @@ object LightningMessageCodecs {
     ("signature" | bytes64) ::
       nodeAnnouncementWitnessCodec).as[NodeAnnouncement]
 
+  val messageFlagsCodec = ("messageFlags" | (ignore(6) :: bool :: constant(bin"1"))).as[ChannelUpdate.MessageFlags]
```

Comment:
```suggestion
  val messageFlagsCodec = ("messageFlags" | (ignore(6) :: bool :: constant(1))).as[ChannelUpdate.MessageFlags]
```

Using scodec's:
```scala
def constant[A: Integral](bits: A*): Codec[Unit] = constant(BitVector(bits: _*))
```

Reply:
Unfortunately not...if you change that all the codec tests break because it actually doesn't include just a bit (I don't dive into why it doesn't behave as expected).